libraries/adaptive-expressions/src/builtinFunctions/subtract.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { MultivariateNumericEvaluator } from './multivariateNumericEvaluator';
+import { ExpressionType } from '../expressionType';
+
+/**
+ * Return the result from subtracting the second number from the first number.
```

Comment:
>second number from the first number [](start = 42, length = 35)

Takes multiple values.

Reply:
Return the result of subtracting all of the numbers one by one.

---
In reply to: [454722349](https://github.com/microsoft/botbuilder-js/pull/2502#discussion_r454722349) [](ancestors = 454722349)

Reply:
This is still not correct.  "Return the result of first argument minus all remaining arguments."

---
In reply to: [456099277](https://github.com/microsoft/botbuilder-js/pull/2502#discussion_r456099277) [](ancestors = 456099277,454722349)